pom.xml

```diff
@@ -69,7 +69,7 @@
         <redirectTestOutputToFile>true</redirectTestOutputToFile>
 
         <!-- versions -->
-        <zookeeper-version>3.6.3</zookeeper-version>
+        <zookeeper-version>3.7.1-SNAPSHOT</zookeeper-version>
```

Comment:
we have to move this to 3.7.1

in 3.7.1 we have fixed a few problems in ZK Embedded, and 3.8.0 is also better

@tisonkun

Reply:
Since @kezhuw resolves CURATOR-596 by #426 I'd cc you here if you're interested in taking over this patch also.

Reply:
I have rebased on current master